### ISSUE TITLE
Inconsistent or missing French filename

### ISSUE THREAD

**Issue by andrew2net** [metanorma/bipm-data-outcomes#33]
The English and French versions of the documents are placed in separate folders, `meetings-en` and ` meetings-fr`, respectively.
For the file [jcrb/meetings-en/meeting-48.yml](https://github.com/metanorma/bipm-data-outcomes/blob/main/jcrb/meetings-en/meeting-48.yml) there is no corresponding French version in the [jcrb/meetings-fr](https://github.com/metanorma/bipm-data-outcomes/tree/main/jcrb/meetings-fr) folder.
There is a French file with a year in its filename [jcrb/meetings-fr/meeting-482024.yml](https://github.com/metanorma/bipm-data-outcomes/blob/main/jcrb/meetings-fr/meeting-482024.yml) but it misses resolutions. Is this a French version of `jcrb/meetings-en/meeting-48.yml`?

**Comment by ronaldtse** [comment]
@andrew2net can you also ping @jmilesBIPM at the BIPM so her colleagues could fix any missing pages?

@jmilesBIPM is this something your team could address? Thanks! (Perhaps Celine or Laurent would also have GitHub accounts?)

**Comment by ronaldtse** [comment]
@andrew2net is this a general problem for all JCRB meetings, or is this only for Meeting 48?

This problem is not affecting the SI Brochure, right? Because this only concerns JCRB Meeting 48?

**Comment by andrew2net** [comment]
> @andrew2net is this a general problem for all JCRB meetings, or is this only for Meeting 48?

It's only for Meeting 48

> This problem is not affecting the SI Brochure, right? Because this only concerns JCRB Meeting 48?

It's affecting the relaton-data-bipm because the repo contains documents from the 3 sources (outcomes, rawbib, and SI Brochure). To update the BIPM dataset the relaton-data-bipm's GHA removes all docs, then fetches new docs from all the 3 sources. If one of the fetching fails, nothing is committed.

**Comment by ronaldtse** [comment]
@andrew2net so how can we fix this issue? We have to handle this asap.

**Comment by andrew2net** [comment]
@ronaldtse we can ignore if a corresponding French file doesn't exist currently. I'll add auto-creating GH issues functionality later.

**Comment by ronaldtse** [comment]
OK let's do that. The key point is to fix the issues first so the BIPM can publish before the deadline!

**Comment by andrew2net** [comment]
@ronaldtse bipm-si-brochure fails https://github.com/relaton/relaton-data-bipm/actions/runs/12383157089/job/34565304478#step:7:1980

**Comment by ronaldtse** [comment]
@andrew2net can we close this now? Thanks.

**Comment by andrew2net** [comment]
> @andrew2net can we close this now? Thanks.

@ronaldtse I don't think so, nothing has changed. Relaton just ignores missing respective files. It seems the file [jcrb/meetings-fr/meeting-482024.yml](https://github.com/metanorma/bipm-data-outcomes/blob/main/jcrb/meetings-fr/meeting-482024.yml) should be renamed to `jcrb/meetings-fr/meeting-48.yml`. The file also misses resolutions.

**Comment by ronaldtse** [comment]
@andrew2net then can you file the issue at bipm-data-importer for @hmdne ? 

According to the URL specified in the file, there are indeed no resolutions listed on that web page. So it is correct not to have resolutions.

**Comment by andrew2net** [comment]
@ronaldtse done
